src/js/view/title.ts

```diff
@@ -3,25 +3,40 @@ import {
     replaceInnerHtml
 } from 'utils/dom';
 import { Browser } from 'environment/environment';
+import type { PlaylistFeedItemWithMedia } from 'parsers/jwparser';
```

Comment:
still unclear of the different types here present in the parsers, but i _think_ i selected the right ones? unclear. i think it might be useful to create a union type for all the possible `itemwithmedia` options to import in cases like the below where we have a generic `item` argument.

thoughts @robwalch ?

Reply:
By the time the view get's it, it should a type in the model (e.g.`model.playlist[model.item]` > `playlist: PlaylistItem[];`). The feed or config input has been parsed and normalized via playlist/playlist.js and item.js (which while untyped should be of type `PlaylistItem`.